handlers/easy_info/__init__.py

```diff
@@ -0,0 +1,12 @@
+from eventhandler import EventHandler
+
+
+class EasyInfoHandler(EventHandler):
+    def on_issue_labeled(self, api, payload):
+        new_label = payload['label']['name']
+
+        if new_label == 'E-Easy':
```

Comment:
It's `E-Easy` on this repo (see #130; <https://github.com/servo/highfive/labels/E-Easy>).

Can you clarify which way would be good (turn everything to lowercase and compare those strings, or the per-repository configuration)?